docs/I18N.md

```diff
@@ -0,0 +1,112 @@
+# Goldstone i18n (internationalization) scheme
+
+## This document describes:
+
+* <a href="#about">How i18n is implemented for the client in Goldstone.
+* <a href="#import">How to import a new .po file into the front-end i18n scheme so it's available as a selection in the client.</a>
+* <a href="#implement">How to implement i18n client-side so that the text strings of your choice are translated to the user-selected language.</a>
+* <a href="#base-html">Special procedures related to having the client translate the django-served base.html file.</a>
+
+### _This document assumes you have installed Goldstone according to the HACKING.md document, and have node/npm installed._
+
+Make sure your dev environment is up to date with the latest npm dependencies with:
+
+`$ npm install`
+
+Please keep in mind that there is no direct relationship between .po files (which are standard for gettext), and JavaScript. So in order for changes or additions to the .po files in `goldstone/static/i18n/po_files` to effect the client rendering, you have to make sure that modifications are made while the Grunt.js task runner is operating by running `grunt watch` in a terminal tab. Alternately you could run `grunt po2json` from the terminal after making changes to a .po file.
```

Comment:
"So in order for changes or additions to the .po files in `.......` to..." ->   "For changes in .po files in `.......` to..."